deploy/helm/sumologic/templates/cleanup/cleanup-job.yaml

```diff
@@ -0,0 +1,71 @@
+{{- if .Values.sumologic.cleanUpEnabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "sumologic.metadata.name.cleanup" . }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
Is it obligatory?

Reply:
I guess not. It was done similarly as setup job and since it's a `pre-delete` hook, it should be fine.

Reply:
I believe we can remove it from the both of the jobs then

Reply:
helm takes care about setting namespace and we do not set namespace for any other template except those two. so lets be consistent and drop setting namespace manually from all of the templates

Reply:
Done